app/src/main/java/com/poupa/vinylmusicplayer/ui/activities/SettingsActivity.java

```diff
@@ -283,9 +322,32 @@ void invalidateSettings() {
                 });
             }
 
-            final ATEColorPreference primaryColorPref = findPreference("primary_color");
+            final TwoStatePreference rememberLastTab = findPreference(PreferenceUtil.REMEMBER_LAST_TAB);
```

Comment:
You need to add those because the UI are not correct? Or otherwise they wont be saved?

Reply:
IIRC yes.